README.md

```diff
@@ -275,4 +275,11 @@ location / {
 	# backend
 	proxy_pass http://127.0.0.1:8000;
 }
-```
\ No newline at end of file
+```
+#### Setting up SMTP server
+The current code works with SSL-based SMTP server using username and password as authentication. Below are the points to note for particular SMTP server:
```

Comment:
Can you change the 2nd sentence to "Here are instructions for setting up particular SMTP servers:"